app/lib/config/config.go

```diff
@@ -33,6 +33,8 @@ type CothoritydConfig struct {
 	Public  string
 	Private string
 	Address network.Address
+	TLSCert string
+	TLSKey  string
```

Comment:
Or, if it is difficult to use byte arrays in toml files: It would be better if (at least) the private key could be stored in a separate file (in a user configurable directory). Then, this would be OK to be left a string.